legal-api/migrations/versions/004f7558063f_second.py

```diff
@@ -1281,6 +1287,7 @@ def downgrade():
         batch_op.drop_index(batch_op.f('ix_addresses_legal_entity_id'))
         batch_op.drop_index(batch_op.f('ix_addresses_change_filing_id'))
         batch_op.drop_index(batch_op.f('ix_addresses_address_type'))
+        batch_op.drop_index(batch_op.f('ix_addresses_alternate_name_id'))
```

Comment:
need to drop `ix_addresses_alternate_name_id ` index for `addresses_history` table too